generate/templates/partials/convert_to_v8.cc

```diff
@@ -61,7 +61,7 @@
     {% if cppClassName == 'Wrapper' %}
       to = {{ cppClassName }}::New({{= parsedName =}});
     {% else %}
-      to = {{ cppClassName }}::New({{= parsedName =}}, false);
+      to = {{ cppClassName }}::New({{= parsedName =}}, {% if selfFreeing %}true{% else %}false{% endif %});
```

Comment:
This is pretty verbose. I think there should be another Combyne filter called `toBool` that just basically returns `!!value`. I'll push that to your branch.